test/mitmproxy/addons/test_view.py

```diff
@@ -546,11 +549,13 @@ def test_configure():
         with pytest.raises(Exception, match="Invalid interception filter"):
             tctx.configure(v, view_filter="~~")
 
-        tctx.configure(v, console_order="method")
+        tctx.configure(v, view_order="method")
         with pytest.raises(Exception, match="Unknown flow order"):
-            tctx.configure(v, console_order="no")
+            tctx.configure(v, view_order="no")
 
-        tctx.configure(v, console_order_reversed=True)
+        tctx.configure(v, view_order_reversed=True)
 
+        console_addon = consoleaddons.ConsoleAddon(tctx.master)
+        tctx.configure(console_addon)
         tctx.configure(v, console_focus_follow=True)
```

Comment:
Sorry, but I can't catch what you mean here, do you mean redeclaring `view_focus_follow` in `consoleaddons`?
This is the case where same option is used by two different addons... How should we deal with this option?